src/Symfony/Bundle/FrameworkBundle/Kernel/MicroKernelTrait.php

```diff
@@ -50,8 +50,8 @@ private function configureContainer(ContainerConfigurator $container, LoaderInte
     {
         $configDir = $this->getConfigDir();
 
-        $container->import($configDir.'/{packages}/*.{php,yaml}');
-        $container->import($configDir.'/{packages}/'.$this->environment.'/*.{php,yaml}');
+        $container->import($configDir.'/{packages}/*.{php,xml,yaml}');
+        $container->import($configDir.'/{packages}/'.$this->environment.'/*.{php,xml,yaml}');
```

Comment:
while at it: php should be last imho (the most dynamic takes over the others)

Reply:
you say i should put .xml last for BC loading order? then routes should follow isnt it.

I wasnt sure this matters from the glob pattern POV :/ but havent tested it either.

Reply:
the list should be yaml,xml,php for services, and php,xml,yaml for routes yes